app/services/attribute_asserter.rb

```diff
@@ -63,6 +65,10 @@ def attribute_getter_function(attr)
     ->(_principal) { decrypted_pii[attr] }
```

Comment:
that's a good point.

I have been relying on the default stringification methods in the `Pii::Attribute` class to determine what string gets sent, because we haven't yet decided whether it is the `raw` or `norm` value. So I am continuing to kick that can down the road by using the implicit value.